.github/workflows/fpga_sdc_reg_test.sh

```diff
@@ -10,3 +10,10 @@ echo -e "FPGA-SDC regression tests";
 
 echo -e "Testing SDC generation with time units";
 run-task fpga_sdc/sdc_time_unit --debug --show_thread_logs
+
+
+# Repgression test to test multi-user enviroment
+# TODO : While restructuring regression test files
+#        move this to separate file
+cp -r */*/fpga_sdc/sdc_time_unit /tmp/
```

Comment:
I would move this test to the ``basic_test``. Also the SDC test case is far more simplified. I suggest to use a Verilog-to-Verification testcase,  such as ``basic_test/full_testbench/configuration_chain``

Reply:
We need to restructure the the regression tests files to improve runtime.
I added in SDC test because that is the fastest for now. 
I do not wanted to increase the runtime of existing tests

Reply:
Basic tests are typically fast to run. That is why I prefer to see them there. SDC tests will be improved once we have access to OpenSTA in a docker image.

Reply:
On the other side, @nachiket may need students to run design flows with verifications. That is why I think the test case have to be changed.

Reply:
Moved

Reply:
Good. Can you switch the test case to a full testbench test case, e.g., ``basic_test/full_testbench/configuration_chain``, which has a better coverage in testing